src/BL/Services/MinioHelper.cs

```diff
@@ -656,5 +657,183 @@ private long CalcularePartSize(long objectSize, long partSize)
+                    Output = result.Output
+                };
+            }
+            catch (Exception ex)
+            {
+                Log.Error(ex, "Error removing user with mc command");
+                return new MinIOOperationResult
+                {
+                    Success = false,
+                    Message = ex.Message
+                };
+            }
+        }
+
+        /// <summary>
+        /// Execute MinIO client command in Docker container (Cross-platform)
+        /// </summary>
+        private async Task<CommandResult> ExecuteMinIOClientCommand(string command)
+        {
+            try
+            {
+                // Adjust container name based on your Docker Compose setup
+                var containerName = _minioSettings.ClientContainerName ?? "minio-client";
+                var dockerCommand = $"docker exec {containerName} {command}";
+
+                var processStartInfo = GetProcessStartInfo(dockerCommand);
+
+                using var process = new Process { StartInfo = processStartInfo };
+                process.Start();
+
+                var output = await process.StandardOutput.ReadToEndAsync();
+                var error = await process.StandardError.ReadToEndAsync();
+
+                await process.WaitForExitAsync();
+
+                Log.Information($"Command: {command}, ExitCode: {process.ExitCode}");
+                Log.Information($"Output: {output}");
+
+                if (!string.IsNullOrEmpty(error))
+                {
+                    Log.Warning($"Error: {error}");
+                }
+
+                return new CommandResult
+                {
+                    ExitCode = process.ExitCode,
+                    Output = output,
+                    Error = error
+                };
+            }
+            catch (Exception ex)
+            {
+                Log.Error(ex, "Error executing MinIO client command");
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Get ProcessStartInfo based on the operating system
+        /// </summary>
+        private ProcessStartInfo GetProcessStartInfo(string command)
+        {
+            if (System.Runtime.InteropServices.RuntimeInformation.IsOSPlatform(System.Runtime.InteropServices.OSPlatform.Windows))
+            {
+                // Windows - use cmd.exe
+                return new ProcessStartInfo
+                {
+                    FileName = "cmd.exe",
+                    Arguments = $"/c \"{command}\"",
+                    RedirectStandardOutput = true,
+                    RedirectStandardError = true,
+                    UseShellExecute = false,
+                    CreateNoWindow = true
+                };
+            }
+            else
+            {
+                // Linux/macOS - use bash
+                return new ProcessStartInfo
+                {
+                    FileName = "/bin/bash",
```

Comment:
Fine for now but it should be noted that this limits the app to running in containers images where bash is available.

e.g. If someone wanted to build their own alpine container for this app, this would stop them.